scripts/util.py

```diff
@@ -3,9 +3,14 @@
 
 """Methods of manipulating the templates repository."""
 
+from asyncio.subprocess import STDOUT
```

Comment:
Can we not do this? I don't like that `STDOUT` somewhere far down in the codebase actually refers to something in the asyncio library, it's just easier to read if this is made clear at the use site IMHO

Reply:
Yes. Will remove. This doesn't belong here. I accidentally allowed the IDE to add this import statement, when I added `stdout` some place else.